SketchUp/Sketchup/Edge.rb

```diff
@@ -35,7 +35,7 @@ class Sketchup::Edge < Sketchup::Drawingelement
   #     UI.messagebox "Failure"
   #   end
   #
-  # @return entities - the edge and entities connected to that edge
+  # @return [Array<Sketchup::Edge, Sketchup::Face>] the edge and entities connected to that edge
```

Comment:
I was wondering about whether it returns anything connected. In my testing it only returned mesh primitives (edge/face), but not dimensions and leader texts. Entity is probably future-save.